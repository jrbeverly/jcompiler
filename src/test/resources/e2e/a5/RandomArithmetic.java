public class RandomArithmetic {

  public RandomArithmetic() {}

  public static int test() {
    System.out.println(1);
    System.out.println((8446 + 17128) * (16691 * 9257 + (10679 - 517) + (18176 - 9787) * (9933 - 8752)));
    System.out.println(17197 / (5479 * - 11314) + - (16336 * 4273) / 11143);
    return 123;
  }

}
