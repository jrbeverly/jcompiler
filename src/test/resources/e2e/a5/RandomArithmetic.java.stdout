1
261581908
-6264
